docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -100,10 +100,40 @@ spec:
       resources:
         requests:
           storage: 1Gi
+---
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+    name: www-0
+    labels:
+        type: local
+spec:
+    storageClassName: my-storage-class
+    capacity:
+        storage: 1Gi
+    accessModes:
+    - ReadWriteOnce
+    hostPath:
+        path: "/tmp/data/www-0"
+---
+kind: PersistentVolume
+apiVersion: v1
+metadata:
+    name: www-1
+    labels:
+        type: local
+spec:
+    storageClassName: my-storage-class
+    capacity:
+        storage: 1Gi
+    accessModes:
+    - ReadWriteOnce
+    hostPath:
+        path: "/tmp/data/www-1"
 ```
 
 ## Pod Identity
-StatefulSet Pods have a unique identity that is comprised of an ordinal, a
+StatefulSet Pods have a unique identity that comprises of an ordinal, a
```

Comment:
revert, the previous is grammatically correct, this is not

Reply:
I will revert the change.
According to dictionary, the use of `of` after the word `comprise` should be avoided. It should be either `is comprised of` or `comprises`.